gpii/node_modules/testing/src/Mocks.js

```diff
@@ -279,7 +279,7 @@ gpii.test.integration.mockSettingsHandler.setImpl = function (that, payload, sol
         var applySettings = function () {
             gpii.test.integration.mockSettingsHandler.applySettings(store, payload.settings);
         };
-        if (that.options.delaying && !payload.options.mockSync) {
+        if (!payload.options.mockSync) {
             // behave, for example, like the real SPI settings handler - claim to apply settings immediately but don't actually do it till later
             gpii.invokeLater(applySettings, 5000);
```

Comment:
If you remove the retrying facility, you should also remove the test feature which fails to apply the settings until after a delay - this entire branch should be removed